glib/src/value.rs

```diff
@@ -205,6 +205,18 @@ pub unsafe trait FromValue<'a>: Sized {
     unsafe fn from_value(value: &'a Value) -> Self;
```

Comment:
Search for `FromValue<'static>` in this file. These should all become `FromValueOwned` AFAICS

Reply:
All of them? the one in ValueType trait definition can't be made as FromValueOwned, it breaks a ton of code

Reply:
All two of them, and especially that one was the one I was thinking of. How does it break?

Reply:
There are different cases:
1 - We need to generate the new trait as part of gir & do a regen (for flags/enums)
2 - we can no longer impl ValueType for `&'static str`
3 - BoxedValue 

Otherwise I fixed the remaining ones

Reply:
For 1- https://github.com/gtk-rs/gir/pull/1254

Reply:
2 - seems expected and correct
3 - what's the problem there?

Reply:
3 - None, i did implement FromOwnedValue for it as well
2 - so, we drop the ValueType impl of it?

Reply:
2 - Yes, it makes no sense. You can't get a `&'static str` from a `Value`